.editorconfig

```diff
@@ -0,0 +1,4 @@
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = crlf
```

Comment:
I'm curious as to why this would be needed... We have developers working on Linux and OSX, and never encountered any issue with the end of line style. They use Sublime Text, which, from what I just read, handles this case transparently.